app/app.component.ts

```diff
@@ -9,20 +9,61 @@ import {Ng2C3} from './library/ng2c3.component';
 @Component({
     selector: 'my-app',
     template: `<div style='width:500px;height:500px'>
-                    <ng2-c3 [data]='_data'></ng2-c3>
-                </div>`
+                    <ng2-c3 [data]='_data' [axis]='_axis' [tooltip]='_tooltip' [chartConfig]="_chartConfig"></ng2-c3>
```

Comment:
Nothing big in difference but it gives consistency if you use same style quotes across.